addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/UniFiControllerConfig.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.unifi.internal;
+
+import java.math.BigDecimal;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ *
+ * The {@link UniFiControllerConfig} encapsulates all the configuration options for an instance of the
+ * {@link UniFiController}.
+ *
+ * @author Matthew Bowman - Initial contribution
+ */
+public class UniFiControllerConfig {
+
+    private String host;
+    private BigDecimal port;
```

Comment:
When you replace `BigDecimal` with `int` you don't have to call `.intValue()` everywhere which will make the code more readable.